SpatialGDK/Documentation/content/toolbar.md

```diff
@@ -74,8 +74,8 @@ The toolbar settings are in **Edit** > **Project Settings** > **SpatialOS Unreal
 
 * **Interop Codegen**
 
-    * **Output path for the interop code generation**</br>
-    Use this to specify the filepath of ICG-generated typebindings file.  If you leave this empty, it defaults to `<GameSourceDir>/<ProjectName>/Generated`.
+    * **Generate Schema for all Supported Classes**</br>
+    **Experimental** Use this to generate schema for all AActor and UActorComponent classes without explicitly tagging them as `SpatialType` with their UCLASS defintion. 
```

Comment:
**Experimental** Use this to generate schema for all AActor and UActorComponent classes (which **don't** extend off of `USceneComponent`) without explicitly tagging them as `SpatialType` with their UCLASS defintion. 

`USceneComponent` is unsupported due to there often being multiple `USceneComponents` on an Actor which currently isn't supported by the GDK.

If you have Actor Components which don't extend off of `USceneComponent` but are attach more than one instance to an Actor, **this can potentially crash your game**. Make sure there are never multiple components of the same type on an Actor. If there are, make sure you modify your actor to only have one or use the `SpatialType` approach and don't mark that Actor Component as `SpatialType`